lsp-types/src/Language/LSP/Protocol/Message/Method.hs

```diff
@@ -29,6 +30,11 @@ import           Unsafe.Coerce
 -- SomeMethod
 ---------------
 
+-- | Is this an "optional" method which servers and clients are allowed to ignore?
+isOptionalMethod :: SomeMethod -> Bool
+-- See https://microsoft.github.io/language-server-protocol/specifications/lsp/3.17/specification/#dollarRequests
+isOptionalMethod m = "$" `isPrefixOf` someMethodToMethodString m
```

Comment:
Where is the `/`?

Reply:
You're right, the spec does say `$/`, although something that started with just a `$` would be pretty confusing.